CMakeLists.txt

```diff
@@ -248,7 +248,7 @@ if (NOT USE_PRE_BUILT_NGRAPH)
     ExternalProject_Add(
         ext_ngraph
         GIT_REPOSITORY https://github.com/NervanaSystems/ngraph
-        GIT_TAG v0.28.0-rc.1
+        GIT_TAG 8e37fbfaeb7ee827746d607d9688dc509a25fe77
```

Comment:
Would it work if you simply specified `tf-bridge-ie-backend` here?

Reply:
Haven't tried